.nycrc

```diff
@@ -1,11 +1,41 @@
 {
     "include": [
-  	  "index.js",
-	  "gpii/node_modules/*/*.js",
-	  "gpii/node_modules/*/src/*.js"
+     "**/*.js",
+     "gpii/node_modules/**/*.js"
     ],
     "exclude": [
-	  "!**/node_modules/"
+      "!**/gpii/node_modules/WindowsUtilities/WindowsUtilities.js",
```

Comment:
Why is the content in `./gpii/node_modules` being excluded?  It's code checked into the repo, it should be tested and covered.

Reply:
I guess you're excluding things that should only be hit by node from the browser tests, I see that now.

Reply:
Sorry, got lost in the similar syntax.  You're definitely excluding node code from node coverage reporting, which should not be the case.